### PR TITLE
add additional path creation command to the vector mask lib

### PR DESCRIPTION
**Pull request by pineapplespatula**
These new commands are used by the new vector mask HUD

### CODE REVIEW

**Comment by iwehrman** [comment]
We now increment the version number as needed during development. According to semver rules, if you add a new API, that means you need to increment the minor version.

File: src/lib/vectorMask.js [hunk 1]
```diff
@@ -69,6 +70,43 @@ define(function (require, exports) {
     };
 
     /**
+     * creates a ciruclar work path with the given bounds descriptor
```
Comment (iwehrman) [inline src/lib/vectorMask.js]: Typo.

**Comment by pineapplespatula** [comment]
Do we bump version numbers more than once per sprint?

**Comment by pineapplespatula** [comment]
@iwehrman :repeat_one:

**Comment by iwehrman** [comment]
Looks good, thanks! The versioning rules will make more sense once we return to sprint work, I think/hope.

**Pull request merged**